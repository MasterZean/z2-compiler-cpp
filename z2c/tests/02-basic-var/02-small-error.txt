val a = -128s8
val b = 128s8
val c = -200s8
val d =200s8

