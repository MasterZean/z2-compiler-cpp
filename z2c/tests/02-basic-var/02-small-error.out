int8 a = -128;
DummyClass(2, 9): error:
	'Small' literal constant too big
DummyClass(3, 9): error:
	'Small' literal constant too big
DummyClass(4, 8): error:
	'Small' literal constant too big
